From: Sriharsha Basavapatna <sriharsha.basavapatna@broadcom.com>
Date: Sun, 9 Oct 2016 09:58:51 +0530
Subject: be2net: Update Copyright string in be_hw.h
Patch-mainline: v4.9-rc1
Git-commit: 77b696cba961bb6e88aeba36253849443f9a4186
References: bsc#1021121 FATE#321675

This patch updates the year and company name in the copyright string
in be_hw.h.

Signed-off-by: Sriharsha Basavapatna <sriharsha.basavapatna@broadcom.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: David Chang <dchang@suse.com>
---
 drivers/net/ethernet/emulex/benet/be_hw.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/ethernet/emulex/benet/be_hw.h
+++ b/drivers/net/ethernet/emulex/benet/be_hw.h
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2005 - 2015 Emulex
+ * Copyright (C) 2005-2016 Broadcom.
  * All rights reserved.
  *
  * This program is free software; you can redistribute it and/or
